pkg/proxy/proxy_test.go

```diff
@@ -155,3 +165,82 @@ func buildMocFormReq(requestFiles map[string]string, distributedPods, ltType str
 	req.Header.Set("Content-Type", request.contentType)
 	return req, nil
 }
+
+func TestProxy_Create(t *testing.T) {
+	for _, tt := range []struct {
+		name                string
+		distributedPods     string
+		failingLine         string
+		loadTestType        string
+		requestFiles        map[string]string
+		expectedCode        int
+		expectedResponse    string
+		expectedContentType string
+	}{
+		{
+			"Valid request, only test file",
+			"10",
+			"",
+			"JMeter",
+			map[string]string{
+				"testFile": "testdata/valid/loadtest.jmx",
+			},
+			http.StatusCreated,
+			"{\"type\":\"JMeter\",\"distributedPods\":10,\"phase\":\"creating\",\"hasEnvVars\":false,\"hasTestData\":false}\n",
+			"application/json; charset=utf-8",
+		},
+		{
+			"Valid request, all files",
+			"10",
+			"",
+			"JMeter",
+			map[string]string{
+				"testFile": "testdata/valid/loadtest.jmx",
+				"testData": "testdata/valid/testdata.csv",
+				"envVars":  "testdata/valid/envvars.csv",
+			},
+			http.StatusCreated,
+			"{\"type\":\"JMeter\",\"distributedPods\":10,\"phase\":\"creating\",\"hasEnvVars\":true,\"hasTestData\":true}\n",
+			"application/json; charset=utf-8",
+		},
+		{
+			"Invalid type",
+			"10",
+			"",
+			"unknown",
+			map[string]string{
+				"testFile": "testdata/valid/loadtest.jmx",
+			},
+			http.StatusBadRequest,
+			"{\"error\":\"invalid LoadTest type\"}\n",
+			"application/json; charset=utf-8",
+		}} {
+		t.Run(tt.name, func(t *testing.T) {
+			var logger = zap.NewNop()
```

Comment:
if you use this

```suggestion
			var logger = zaptest.NewLogger(t)
```

you will get logger output if test fails